app/src/main/java/com/pvp/app/ui/screen/feedback/FeedbackForm.kt

```diff
@@ -0,0 +1,146 @@
+package com.pvp.app.ui.screen.feedback
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Star
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.pvp.app.R
+import com.pvp.app.ui.common.Button
+import com.pvp.app.ui.common.EditableTextItem
+import com.pvp.app.ui.common.TabSelector
+
+@Composable
+fun FeedbackCreationDialog(
+    isOpen: Boolean,
+    onClose: () -> Unit
+) {
+    if (!isOpen) {
+        return
+    }
+
+    Dialog(onDismissRequest = onClose) {
+        FeedbackForm(onCreate = onClose)
+    }
+}
+
+@Composable
+fun FeedbackForm(
+    model: FeedbackViewModel = hiltViewModel(),
+    onCreate: () -> Unit
+) {
+    var bug by remember { mutableStateOf(true) }
+    var description by remember { mutableStateOf("") }
+    var rating by remember { mutableStateOf(0) }
+
+    val isFormValid by remember { derivedStateOf { description.isNotEmpty() } }
+    val localeCreate = stringResource(R.string.action_create)
```

Comment:
If it is used only in one place, could drop the variable and call stringResource in parameter spot, like other ones.